public/index.html

```diff
@@ -3,19 +3,17 @@
 <head>
     <meta charset="utf-8">
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
-    <title>Page Title</title>
+    <title>Proxy Page</title>
     <meta name="viewport" content="width=device-width, initial-scale=1">
     <link rel="stylesheet" type="text/css" media="screen" href="http://localhost:3016/styles.css">
+    <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.7.2/css/all.css" integrity="sha384-fnmOCqbTlWIlj8LyTjo7mOUStjsKC4pOpQbqyi7RrhN7udi9RwhKkMHpvLbHG9Sr" crossorigin="anonymous">
 </head>
 <body>
-    <h1>Proxy Server</h1>
-    <div id="app"></div>
+    <div id="Details"></div>
+    <div id="events"></div>
     <script src="/lib/react.development.js"></script>
     <script src="/lib/react-dom.development.js"></script>
-    <!-- <script src="http://localhost:3016/client/dist/bundle.js"></script> -->
+    <script src="http://localhost:3014/bundle.js"></script>
     <script src="http://localhost:3016/bundle.js"></script>
-    <!-- <script>
```

Comment:
Awesome that you removed the actual path to the bundle.js file.  Can you tell me why you don't need to use that?

Hint: it has something to do with `express.static()`